recipe/meta.yaml

```
@@ -1,34 +1,38 @@
 {% set name = "pychrono" %}
 {% set version = "4.0.0" %}
-{% set build = 2 %}
-{% set sha256 = "803b5089e48334771f652f5bec79f95f2a42cfb02d2ffbaf8ad9e5dca2ba1886" %}
+{% set build = 3 %}
+{% set sha256 = "7ee0d972cf46bb5ea2342b6b38c77cce4dc502562515983ed7551152db7c219d" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/projectchrono/chrono/archive/{{ version }}.tar.gz
+  url: https://github.com/projectchrono/chrono/tarball/develop
+  fn: develop.tar.gz
   sha256: {{ sha256 }}
 
 build:
   number: {{ build }}
-  skip: true  # [not linux]
+  skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
-    - swig 3.0.12
+    - swig 4.0.1
   host:
-    - python
+    - python 3
```

Pinning the Python version like this will make bad things happen. My other comment for how to do it correctly.
```suggestion
    - python
```

---

To add, you should ask the bot to rerender after fixing this.